setup.cfg

```diff
@@ -0,0 +1,65 @@
+[metadata]
```

Comment:
this required? they conflict with https://github.com/jpadilla/pyjwt/pull/995

Reply:
Removed